lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -81,6 +83,36 @@ class DetailsRenderer {
     element.appendChild(items);
     return element;
   }
+
+  /**
+   * @param {!CardsDetailsJSON} details
```

Comment:
closure is the worst but not for this reason. The parameter can be one of two different things, so... :P